### PR TITLE
Changes made to Wide and deep vertex pipelines notebook

### PR DESCRIPTION
**Pull request by bhupana**
PR problem description : Extra components. Added Libraries and Deleted unnecessary code components from code now and rectified the error "The DAG failed because some tasks failed.

1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb [hunk 1]
```diff
@@ -29,7 +29,7 @@
         "id": "mThXALJl9Yue"
```
Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>Put Learn more about on separate line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>In Objective, why does the user train 2 models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>Add Dataflow to the bullet list</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>Add short description for what each helper function does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>The description of arguments you can define.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>The description of arguments you can define.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Comment (andrewferlitsch) [inline notebooks/official/tabular_workflows/wide_and_deep_on_vertex_pipelines.ipynb:29]: <p>parameter. you -&gt; parameter. You</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

**Review by ivanmkc — APPROVED** [review]
LGTM

**Comment by ivanmkc** [comment]
@andrewferlitsch It seems like this pipeline notebook has unrelated (?) execution issues: https://pantheon.corp.google.com/vertex-ai/locations/us-central1/pipelines/runs/wide-and-deep-4hl4cl1t?project=python-docs-samples-tests

**Comment by andrewferlitsch** [comment]
Step #5: Step #4: Creating PipelineJob
Step #5: Step #4: PipelineJob created. Resource name: projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa
Step #5: Step #4: To use this PipelineJob in another session:
Step #5: Step #4: pipeline_job = aiplatform.PipelineJob.get('projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa')
Step #5: Step #4: View Pipeline Job:
Step #5: Step #4: https://console.cloud.google.com/vertex-ai/locations/us-central1/pipelines/runs/wide-and-deep-isk1u1qa?project=1012616486416
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: Step #4: PipelineJob projects/1012616486416/locations/us-central1/pipelineJobs/wide-and-deep-isk1u1qa current state:
Step #5: Step #4: PipelineState.PIPELINE_STATE_RUNNING
Step #5: 
Step #5: 
Step #5: === END RESULTS===
Step #5: 
Step #5: Cumulative notebook duration: 00:15:03
Step #5:     raise RuntimeError("Notebook failures detected. See logs for details")
Step #5: RuntimeError: Notebook failures detected. See logs for details

**Comment by ivanmkc** [comment]
@andrewferlitsch Since Sudarshan won't work on this notebook anymore, you'll need to branch off of his branch and create a new PR.

**Comment by andrewferlitsch** [comment]
Pending determination

**Comment by andrewferlitsch** [comment]
SpringML no longer a contractor and code does not resolve the failed DAG issue. Closing the PR.

**Pull request closed without merging**